src/applications/mhv-medical-records/containers/SettingsPage.jsx

```diff
@@ -172,7 +174,9 @@ const SettingsPage = () => {
         modalTitle={title}
         onCloseEvent={() => {
           handleCloseModal();
-          sendDataDogAction(`Close opt ${isSharing ? 'out' : 'in'} modal`);
+          sendDataDogAction(
+            `Close ${isSharing ? 'Opt out - Modal' : 'Opt in - Modal'}`,
+          );
```

Comment:
I believe these tags are supposed to be applied to the `primaryButtonText` and `secondaryButtonText` variables just above.